_episodes/conclusion.md

```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "How can I assess the FAIRness of my data?"
+- "What do I do now to improve the FAIRness of my data?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Relate each FAIRification practice with FAIR principles."
+- "Know about available tools/resources for assessing FAIRness of your data."
+- "Understand the next steps you can take to improve FAIRness of your data."
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "FAIRification practices are about documentation, metadata, file formats, access to data,
+persistent identifiers, and data licenses."
+- "FAIR will change the way we do science."
+- "FAIR brings all the stakeholders together."
```

Comment:
I guess here you want to say something about how the FAIR principles affect everyone, from top to bottom, how about:

```suggestion
- "The FAIR principles promote maximum use of research data, involving all the stakeholders from data producers to funding agencies."
```